extensions/v1/kiali/spec.yaml

```diff
@@ -8,9 +8,14 @@ longDescription: |
 
   A service mesh can now provide these services on a platform level and frees the application writers from those tasks. Routing decisions are done at the mesh level.
 
-  Kiali works with Istio to visualise the service mesh topology, features like circuit breakers or request rates.
+  Kiali works with Istio to visualize the service mesh topology, features like circuit breakers or request rates.
 
   Kiali also includes Jaeger Tracing to provide distributed tracing out of the box.
+
+  To try, run the following commands
```

Comment:
Maybe we should add a README for each extension  to include information like this

Reply:
I feel (ideally) we should have a dedicated section for each extension that lets providers either document next steps for users after they install the extension, i.e., steps to get to Kiali console OR a link to getting started documents for extensions like Flagger (nothing visual).

OperatorHub has a links sections which could be a compromise with document and getting started documents.